### DISCUSSION TITLE
custom css relocation

### DISCUSSION THREAD

**Discussion by piyoki** [thegeeklab/hugo-geekdoc#232] [Q&A] [answered]
Hi,

I noticed that the default `custom.css` file is set to be located in `static/custom.css`. I wonder if there is any way I can load my custom css file from `static/css/custom.css`?

**Comment by xoxys** [comment, marked as answer]
Hi,

yes you can overwrite https://github.com/thegeeklab/hugo-geekdoc/blob/main/layouts/partials/head/others.html#L51-L60 or add your own CSS file to e.g. https://github.com/thegeeklab/hugo-geekdoc/blob/main/layouts/partials/head/custom.html
